docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+# Contributing
+
+When contributing to this repository please adhere to the guidelines below.
+
+# Mandatory Requirments
+
+- Every line of code should be written profesionally and clearly
+- Before creating a PR proofread your work and make sure it is ready for review
+- Create your commands in the category relative to the command
+- Test your build locally before submitting anything for review
+
+# Writing Guidelines
+
+All guidelines created for this repository must be followed at all times to ensure the smooth operation of the bot not only for the developers but also for the community.
+
+Please refer to the listings below to achieve this.
+
+- Refrain from any use of exclamation points (!) unless it is for a specific reason e.g. (warnings)
+- Please use the proper capitilisation and names for 3rd party services e.g. (A32NX and not a32nx)
+- Ensure that all spelling,punctuation and grammar are correct
+- Ask a Bot-Developer for help if you are unsure or stuck with anything when writing your command
+- Send a message in #Discord-Bot when beginning work on a command or if you are interested in adding a command that has not yet been identified
+
+# Creating a Pull request (PR)
+
+Reminder: all PRs are pushed into 'staging' before being merged into the 'master' branch
+
+- 1. Fork the repository
+- 2. Clone to your local system using your IDE of choice
+- 3. Create a new branch from 'staging' and name it according to the command you are making e.g. (feat: adds VNAV command, fix: fixes grammar in VNAV)
+- 4. Develop or edit the command that you are working on
+- 5. Test your build locally (Instructions below)
+- 6. Create a PR to merge your command into staging (Template at bottom)
+
+# Testing your build
+
+### node and npm install
+
+- 1. [install node](https://nodejs.org/en/download/), npm is bundled with the download.
+- 2. Open a command prompt or terminal in your repo directory and type `npm install`.
```

Comment:
Development of the bot also requires node-ts, as the source is typescript

Reply:
isn't that the same as the FBW bot? but we just don't do it.